soccer/gameplay/tactics/stopped/circle_near_ball.py

```diff
@@ -131,7 +131,10 @@ def execute_completed(self):
                 num_robots += 1
 
         i = 0
-        for pt in self.get_circle_points(num_robots):
+        for pt in self.get_circle_points(6 if num_robots == 0 else num_robots):
```

Comment:
If we don't know how many robots the tactic will have it defaults to 0, hence the "if num_robots == 0". When this is the case, we don't want the tactic to assign any robots so we give it 6 impossible points so that robots are automatically reassigned to the best point when they are actually added to the tactic and proper points are calculated.